src/gwt/src/org/rstudio/core/client/widget/VirtualizedDataGrid.java

```diff
@@ -179,7 +179,7 @@ public void redrawIfNecessary()
       
       if (changed)
       {
-         redraw();
+         super.redraw();
```

Comment:
Why this change?

Reply:
I reorganized the code to make this more clear; the intention was to avoid an unnecessary second call to `updateActiveRows()`.